smac/tae/execute_ta_run.py

```diff
@@ -159,17 +185,21 @@ def start(self, config:Configuration,
                                     "exit. The last incumbent can be found "
                                     "in the trajectory-file.")
 
-        # update SMAC stats
-        self.stats.ta_runs += 1
-        self.stats.ta_time_used += float(runtime)
-
         if self.run_obj == "runtime":
-            if status != StatusType.SUCCESS:
-                cost = cutoff * self.par_factor
-            else:
+            if cutoff is not None and runtime > self.par_factor * cutoff:
+                self.logger.warn("Returned running time is larger "
+                                 "than 2 times the passed cutoff time. "
```

Comment:
This warning seems to be incorrect. This if-statement is true when the runtime is larger that par\*cutoff and not 2\*cutoff.